ClientSide(FrontEnd)/Consumer_Side/src/components/Displayproduct.jsx

```diff
@@ -26,6 +26,16 @@ const Displayproduct = () => {
         })
     }
 
+
+    const categaryProductData = () => {
+        fetch(`http://localhost:5050/search/${inputValue}`).then((res)=>{
+            return res.json();
+        }).then((res)=>{
+            setPotteryData(res);
+            console.log(res);
```

Comment:
Do not expose `console.log` to the end user.  Incase if it is necessary then add logs only in the developer mode